tests/test_api.py

```diff
@@ -25,6 +25,59 @@ def test_get_study_summaries(self) -> None:
         study_summaries = json.loads(body)["study_summaries"]
         self.assertEqual(len(study_summaries), 2)
 
+    def test_get_study_details(self) -> None:
```

Comment:
Could you separate test cases to avoid [Assertion Roulette](http://xunitpatterns.com/Assertion%20Roulette.html)?